keyboards/dumbpad/keymaps/imchipwood/config.h

```diff
@@ -0,0 +1,2 @@
+#include "../../config.h"
```

Comment:
This is not needed, and is automagically handled by the make system. 

However: 
```suggestion
#pragma once
```